server/rest.py

```diff
@@ -97,6 +105,7 @@ def _parseTestParams(params):
         .errorResponse('ID was invalid.')
         .errorResponse('Read access was denied for the item.', 403)
     )
+    @access.cookie
```

Comment:
Why do we need cookie access here? The metadata endpoint should normally only be called via Ajax, which will be able to set headers.

The image-returning endpoints might be fetched by the DOM (e.g. in ``<img src=``), so they must allow cookie auth.